tests/FtpClientTest.php

```diff
@@ -324,7 +324,7 @@ public function testCopyFromLocalWithFileSource()
         }
 
         $this->assertTrue($this->getFtpClientInstance()->copyFromLocal($tempFile, INITIAL_DIR));
-        $this->getFtpClientInstance()->removeFile(INITIAL_DIR . "/$tempFile");
+        $this->getFtpClientInstance()->removeFile(INITIAL_DIR . "/" . basename($tempFile));
```

Comment:
Thank you, I've already fixed that before and I've forgotten to push the changes. however, I've merged a PR with the fixes in #9.